bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,43 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    class << self
+      attr_accessor :machine_id
+    end
+
+    I18N_METRICS_NAMESPACE = 'I18n'.freeze
+
+    def self.report_runtime(addtl_dimensions, value)
+      log_metric(:Runtime, addtl_dimensions, value)
+    end
```

Comment:
I would recommend using `block` (`yield`) for capturing `value`:
```suggestion
    # Using: `I18n::Metrics.report_runtime([{name: "dim1", value: 1}, {name: "dim2", value: 2}]) { value }`
    def self.report_runtime(addtl_dimensions)
      log_metric(:Runtime, addtl_dimensions, yield)
    end
```

Reply:
oh interesting, why?

Reply:
I think it will make the method more readable and understandable. For example, if you provide 10 `addtl_dimensions`, the `value` argument becomes less noticeable against the background of those arguments.

Reply:
Or we could provide the `value` argument first before the `addtl_dimensions`: `def self.report_runtime(value, addtl_dimensions)`

Reply:
It's just a suggestion

Reply:
Yeah, I think of `addtl_dimensions` as the final argument